lib/subghz/protocols/bett.c

```diff
@@ -242,7 +242,6 @@ void subghz_protocol_decoder_bett_feed(void* context, bool level, uint32_t durat
         if(!level) {
             if(DURATION_DIFF(duration, subghz_protocol_bett_const.te_short * 44) <
                (subghz_protocol_bett_const.te_delta * 15)) {
-                instance->decoder.parser_step = BETTDecoderStepSaveDuration;
                 if(instance->decoder.decode_count_bit ==
```

Comment:
why did you remove this line? Have you seen how the protocol itself looks like? she is needed there

Reply:
l232 switch(instance->decoder.parser_step) {
l241 case BETTDecoderStepSaveDuration:
l245 instance->decoder.parser_step = BETTDecoderStepSaveDuration;

so it is already set at the good value, no ?